### PR TITLE
As/name attribute autofill

### PR DESCRIPTION
**Pull request by soncuteanca**
# Description

Completed the name autofill test

## Bugzilla bug ID

**ID: 1895878**
**Link: https://testrail.stage.mozaws.net/index.php?/cases/view/122356**

## Type of change

Please delete options that are not relevant.

- [x ] New Test

# How does this resolve / make progress on that bug?
Completed.

Please describe the progress or significance with respect to the bug listed above.

# Screenshots / Explanations

Please upload any relevant media or add a relevant description with respect to the bug listed above.

# Comments / Concerns

Please add a short blurb about any comments or concerns that this change might cause.

### CODE REVIEW

File: modules/page_object_autofill_test_basic.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def fill_input_element(self, ba: BrowserActions, field_name: str, term: str):
         ba.clear_and_fill(web_elem, term, press_enter=False)
 
     def click_form_button(self, field_name):
-        self.get_element("submit-button", field_name).click()
+        self.get_element("submit-button", labels=[field_name]).click()
```
Comment (sarinali) [inline modules/page_object_autofill_test_basic.py:63]: would prefer a type annotation on `field_name` like `field_name: str` or something!

**Review by sarinali — COMMENTED** [review]
minor change

File: modules/page_object_autofill_test_basic.py (hunk 1 above)
Comment (soncuteanca) [reply modules/page_object_autofill_test_basic.py:63]: It is done

**Review by ben-c-at-moz — APPROVED** [review]
This is good to go!

**Pull request merged**